ci-operator/config/openshift/release/openshift-release-master__ci-4.11-upgrade-from-stable-4.10.yaml

```diff
@@ -38,7 +38,7 @@ tests:
 - as: e2e-azure-upgrade
   interval: 24h
   steps:
-    cluster_profile: azure4
+    cluster_profile: azure-2
```

Comment:
You also need (for both aws and azure) to set the `BASE_DOMAIN` param, in each test, e.g:

https://github.com/openshift/release/blob/325a86549b159a428eddac99edca7d84e2cd82bb/ci-operator/config/openshift/release/openshift-release-master__ci-4.10.yaml#L93

and 

https://github.com/openshift/release/blob/325a86549b159a428eddac99edca7d84e2cd82bb/ci-operator/config/openshift/release/openshift-release-master__ci-4.10.yaml#L108

Reply:
I think I got them, but the generated file doesn't appear to have updated after running make jobs

Reply:
That's fine.